wandb/data_types.py

```diff
@@ -30,18 +30,22 @@ def val_to_json(key, val, mode="summary", step=None):
             converted = util.convert_plots(val)
     elif util.is_plotly_typename(typename):
         converted = util.convert_plots(val)
-    if isinstance(val, Image):
+    if isinstance(val, Image) or isinstance(val, Audio):
         val = [val]
 
     if isinstance(val, collections.Sequence) and len(val) > 0:
         is_image = [isinstance(v, Image) for v in val]
-        if all(is_image):
+        is_audio = [isinstance(v, Audio) for v in val]
```

Comment:
Are None's allowed / possible here? Are they considered Images / Audios?

Reply:
If any is None, it throws the error "Mixed media types in the same list aren't supported."  

@shawnlewis , @vanpelt  Is that the desired behavior, or should we filter out None from that array?

Reply:
I'm not sure... can you think of a use case where you'd want to send None?

Reply:
In a History where not all time steps have Images